src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -142,10 +142,11 @@ internal static unsafe bool TryReadFile(Stream stream, byte* buffer, long start,
             return true;
         }
 
+#pragma warning disable BCL0015 
         private static unsafe class NativeMethods
         {
             // API sets available on modern platforms:
-            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-2-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
l1-1-0.dll is the lowest APIset that exposes this method. Using it guarantees that windows would have authored the forwarders for win7 and our api-set package would have the necessary down level forwarder dll that will route this call to kernel32.dll making it work on win7.  Using 1-2-0 would mean it will work on all windows editions where the api set is available on-box and that will most likely not include win7.